swim/memberlist.go

```diff
@@ -280,6 +305,17 @@ func (m *memberlist) Update(changes []Change) (applied []Change) {
 	if len(applied) > 0 {
 		oldChecksum := m.Checksum()
 		m.ComputeChecksum()
+
+		for _, change := range applied {
+			if change.Source != m.node.address {
+				m.logger.WithFields(bark.Fields{
+					"remote": change.Source,
+					// changes in ringpop go do not have an id.
+					// "updateId": change.id,
```

Comment:
Are changes in ringpop going to have an `updateId` in the immediate future? If not, this commented-out line seems like code smell.